databricks/sdk/config.py

```diff
@@ -473,3 +498,36 @@ def deep_copy(self):
         """Creates a deep copy of the config object.
         """
         return copy.deepcopy(self)
+
+    def get_compute(self):
+        return Compute(self)
+
+
+class Compute:
+    serverless: bool = False
+    cluster_id: Optional[str] = None
+
+    def __init__(self, cfg: Config):
+        if cfg.compute is not None and cfg.cluster_id is not None:
+            raise ValueError("Cannot specify both compute and cluster id in the config file")
+        elif cfg.compute is not None:
+            if cfg.compute == "serverless":
+                self.serverless = True
+            elif cfg.compute.startswith("cluster:"):
+                self.cluster_id = cfg.compute.split("cluster:")[1]
+            else:
+                raise ValueError("compute must be either 'serverless' or 'cluster:<cluster_id>'")
+        elif cfg.cluster_id is not None:
+            self.cluster_id = cfg.cluster_id
+
```

Comment:
Looks like this is cross-populating values, i.e., should Compute be populated by the value of `DATABRICKS_CLUSTER_ID`? 
I thought we decided it would not, right?

Reply:
it's not really cross-population, `Compute` is an adapter object to make it easier to get `serverless` and `cluster_id` values from the raw `.compute` and `.cluster_id` attributes. `config.compute` and `config.cluster_id` are not changed and remain the same as specified by the user